deployments/nginx-behind-envoy-deployment-no-qat.yaml

```diff
@@ -39,7 +39,6 @@ data:
               stat_prefix: ingress_http
               route_config:
                 name: local_route
-                require_ssl: all
```

Comment:
Do you think this should be replaced with `require_tls` or removed completely?

Reply:
I thought about `require_tls` but decided to drop this completely. To me it looks this is a `route.VirtualHost` message anyways so it's not in the right place and, also, our `return 200` config does not specify this option...